kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/ExtensibleBot.kt

```diff
@@ -266,6 +266,10 @@ public open class ExtensibleBot(
     public open suspend fun addExtension(builder: () -> Extension) {
         val extensionObj = builder.invoke()
 
+        if (extensions.contains(extensionObj.name)) {
+            logger.warn { "Using duplicate extension name: ${extensionObj.name}" }
```

Comment:
This should use `logger.error` instead and refuse to load the extension - duplicate extension names are _breaking_, as the already existing extension will no longer be able to be referred to.

Reply:
In case you only look at emails: Edited the original comment

Reply:
Yeah OK, I wasn't sure if you'd want the default behaviour changing, pushing now

Reply:
The push is good, but the message isn't clear enough. Maybe something like...

`Extension with duplicate name ${extensionObj.name} registered - unloading previously registered extension.`